AUFGABE.md

```diff
@@ -0,0 +1,63 @@
+
+# Die Aufgabe dieser Webseite
+
+Die Website openstreetmap.de ist das Portal der deutschen OSM-Community in die
+Welt von OpenStreetMap. Es richtet sich an die allgemeine Öffentlichkeit,
+insbesondere in Deutschland bzw. an deutschsprachige Leser. Die Webseite soll
+OSM in seiner ganzen Breite darstellen.
+
+Die Website stellt darüber hinaus auch die Aktivitäten der deutschen
+OSM-Community innerhalb und außerhalb des FOSSGIS e.V. dar.
+
+Die Website ist außerdem auch ein Anlaufpunkt für die deutschsprachige
+OSM-Community.
+
+## Zielgruppen
+
+Die Webseiten richten sich an
+
+1. Interessenten, die OSM noch überhaupt nicht kennen oder nur eine grobe Idee
+   haben. Wenn sie (z.B. über eine Suche) auf diese Webseiten kommen, sollen
+   sie Einführungstexte zu den wichtigsten Themen direkt auf der Seite finden
+   und häufige Fragen direkt beantwortet bekommen. Sie sollen auch angeregt
+   werden, bei OSM mitzumachen und dazu entsprechende Informationen finden.
+2. Einsteiger, die OSM schon kennen, aber nach weiteren Informationen suchen.
+   Sie sollen Informationen direkt auf der Seite finden, oder Links ins Wiki
+   oder zu anderen Seiten, die ein Thema vertieft darstellen.
+3. Mapper, also die OSM-Community für ihre täglichen Bedürfnisse. Hier gibt es
+   Antworten auf FAQs, Hinweise auf Tools und Möglichkeiten mit anderen Mappern
+   in Kontakt zu kommen oder sich über Aktivitäten in Deutschland zu
+   informieren.
+4. Nutzer, also alle, die OSM niedrigschwellig nutzen wollen, zum Beispiel
+   einfach mal auf der OSM-Karte etwas anschauen oder eine Route von A nach B
+   finden wollen.
+
+## Sprache
+
+Die Webseiten werden in deutscher Sprache verfasst. Wo sinnvoll wird bei
+Verweisen auf externe Webseiten (z.B. zum Wiki) auf die deutsche Version einer
+Seite verwiesen.
+
+Der Stil ist in der Regel informell, wir Duzen unsere Leser.
```

Comment:
"duzen" ist hier ein Verb und wird daher kleingeschrieben, auch wenn die Du-Anrede in Briefen groß geschrieben werden darf (früher musste).